parsl/executors/high_throughput/executor.py

```diff
@@ -629,8 +629,8 @@ def submit(self, func, resource_specification, *args, **kwargs):
         """Submits work to the outgoing_q.
 
         The outgoing_q is an external process listens on this
-        queue for new work. This method behaves like a
-        submit call as described here `Python docs: <https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor>`_
+        queue for new work. This method behaves like a submit call as described here `Python docs: <https://docs.python.org/3/
```

Comment:
I was surprised that splitting this URL over two lines worked, but apparently it does - I went into the documentation build for this PR, which is available in the checks section of your PR, and then found the `submit` method for High Throughput Executor, and checked that this link works there.

Reply:
I am glad it worked!. I'd be looking through the issues for other tasks I can work on. Thank you!